### PR TITLE
1.9 Docs improve reference structure

### PR DESCRIPTION
**Pull request by NigelBreslaw**

### CODE REVIEW

File: docs/astro/src/content/docs/guide/development/translations.mdx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ---
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+// cSpell: ignore msgfmt msgctx msginit poedit Lokalize Transifex
```
Comment (tronical) [inline docs/astro/src/content/docs/guide/development/translations.mdx:3]: Ahh, that's where to place the cSpell "spell" in MDX files. TIL :)

**Review by tronical — APPROVED** [review]

File: docs/astro/src/content/docs/guide/development/translations.mdx (hunk 1 above)
Comment (NigelBreslaw) [reply docs/astro/src/content/docs/guide/development/translations.mdx:3]: Also did you notice how to turn on or off the next / previous buttons at the bottom of certain pages? It's a shame that behaviour isn't automatic but the grouping a 3rd party plugin, not inbuilt Starlight.

Comment (tronical) [reply docs/astro/src/content/docs/guide/development/translations.mdx:3]: I thought that’s kind of neat ;-)

File: docs/astro/astro.config.mjs [hunk 2]
```diff
@@ -172,103 +172,150 @@ export default defineConfig({
...
-                                            "reference/std-widgets/switch",
-                                            "reference/std-widgets/tabwidget",
-                                            "reference/std-widgets/textedit",
-                                            "reference/std-widgets/timepicker",
+                                            {
+                                                label: "Overview",
+                                                slug: "reference/keyboard-input/overview",
+                                            },
+                                            {
+                                                label: "HorizontalLayout",
```
Comment (tronical) [inline docs/astro/astro.config.mjs:224]: This seems wrong, the layouts duplicated under keyboard input.

**Pull request merged**